SonarQube.Common/SarifUtilities.cs

```diff
@@ -0,0 +1,91 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SarifUtilities.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace SonarQube.Common
+{
+    public class SarifUtilities
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "protected" constructor or the "static" keyword to the class declaration. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=csharpsquid%3AS1118)